lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -1575,6 +1655,8 @@ def compare(self, os):
             differences.add('restart_policy_delay', parameter=self.restart_policy_delay, active=os.restart_policy_delay)
         if self.restart_policy_window is not None and self.restart_policy_window != os.restart_policy_window:
             differences.add('restart_policy_window', parameter=self.restart_policy_window, active=os.restart_policy_window)
+        if self.has_rollback_config_changed(os.rollback_config):
```

Comment:
```suggestion
        if self.rollback_config is not None and self.has_rollback_config_changed(os.rollback_config):
```

Reply:
Good catch, actually I missed to default `self.rollback`-config to None if it was not defined. Fixed now.